### ISSUE TITLE
Development

### ISSUE THREAD

**Issue by jdevalk** [billerickson/Term-Menu-Order#6]
Hey Bill,

The plugin page points here for support, do you want to get rid of the plugin? We still use it, so I’d be happy to pick it up and fix the issues that it has.

**Comment by billerickson** [comment]
Yes, please take it over! I don't have the time to commit to the maintenance of free plugins anymore.

**Comment by jdoubleu** [comment]
@jdevalk do you provide this plugin on github or in the wordpress plugin repository.
